app/frontend/javascript/controllers/dropdown_controller.js

```diff
@@ -1,31 +1,45 @@
 import { Controller } from "@hotwired/stimulus";
```

Comment:
were these changes to the nav dropdowns mixed in here on purpose? did you have to change the stimulus controller for something on the faq's page?